osu.Game.Rulesets.Catch/Mods/CatchModRelax.cs

```diff
@@ -1,12 +1,53 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Input;
+using osu.Framework.Input.Bindings;
+using osu.Framework.Input.Events;
+using osu.Game.Rulesets.Catch.Objects;
+using osu.Game.Rulesets.Catch.UI;
 using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.UI;
+using osuTK;
+using System;
 
 namespace osu.Game.Rulesets.Catch.Mods
 {
-    public class CatchModRelax : ModRelax
+    public class CatchModRelax : ModRelax, IApplicableToDrawableRuleset<CatchHitObject>
     {
         public override string Description => @"Use the mouse to control the catcher.";
+
+        public void ApplyToDrawableRuleset(DrawableRuleset<CatchHitObject> drawableRuleset) =>
+            (drawableRuleset.Playfield.Parent as Container).Add(new CatchModRelaxHelper(drawableRuleset.Playfield as CatchPlayfield));
```

Comment:
directly cast to `Container<Drawable>` instead of as operator.
you might want to ensure it's not a CompositeDrawable.